src/App/Command/GithubReviewReportCommand.php

```diff
@@ -101,60 +102,29 @@ private function assertInput(InputInterface $input, OutputInterface $output): bo
-        $insiders = $_ENV['REVIEW_INSIDER'] ? explode(',', $_ENV['REVIEW_INSIDER']) : $this->github->getMaintainers();
-
-        foreach ($pullRequests as $pullRequest) {
-            foreach ($pullRequest['reviews']['edges'] as $review) {
-                $review = $review['node'];
-                $date = date('Y-m-d', strtotime($review['createdAt']));
-                $reviewer = $review['author']['login'];
-                $state = $review['state'];
-                $authorIsInsider = in_array($pullRequest['author']['login'], $insiders);
-                $reviewerIsMaintainer = in_array($reviewer, $this->github->getMaintainers());
-
-                if (!$reviewerIsMaintainer) {
-                    continue;
-                }
-                if ($state == 'DISMISSED') {
-                    continue;
-                }
-                if ($date < $this->dateStart) {
-                    continue;
-                }
-                if ($date > $this->dateEnd) {
-                    continue;
-                }
+        $insiders = isset($_ENV['REVIEW_INSIDER']) ? explode(',', $_ENV['REVIEW_INSIDER']) : $this->github->getMaintainers();
 
-                if ($input->getOption('byDate')) {
-                    // Review by date
-                    if (!in_array($reviewer, $reviewsDateAuthor)) {
-                        $reviewsDateAuthor[] = $reviewer;
-                    }
-                    if (!isset($reviewsDate[$date])) {
-                        $reviewsDate[$date] = [];
-                    }
-                    if (!isset($reviewsDate[$date][$reviewer])) {
-                        $reviewsDate[$date][$reviewer] = 0;
-                    }
-                    ++$reviewsDate[$date][$reviewer];
-                } else {
-                    // Review by author
-                    if (!isset($reviewsAuthor[$reviewer])) {
-                        $reviewsAuthor[$reviewer] = [
-                            'ALL' => 0,
-                            'COMMENTED' => 0,
-                            'APPROVED' => 0,
-                            'CHANGES_REQUESTED' => 0,
-                            'INSIDE' => 0,
-                            'OUTSIDE' => 0,
-                        ];
-                    }
-                    ++$reviewsAuthor[$reviewer]['ALL'];
-                    ++$reviewsAuthor[$reviewer][$state];
-                    ++$reviewsAuthor[$reviewer][$authorIsInsider ? 'INSIDE' : 'OUTSIDE'];
+        $maintainers = $this->github->getMaintainers();
+
+        foreach ($maintainers as $maintainer) {
+            $reviewsFilter = new ReviewFilter(
+                $maintainer,
+                \DateTime::createFromFormat('Y-m-d', $this->dateStart),
+                \DateTime::createFromFormat('Y-m-d', $this->dateEnd)
+            );
+            $reviewContributionsByOrganization = $this->github->getReviewsContributions('MDEyOk9yZ2FuaXphdGlvbjI4MTU2OTY=', $reviewsFilter);
```

Comment:
A token ?

Reply:
I will add a comment to explain it's the Organization ID